src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBannerCircular.vue

```diff
@@ -225,6 +249,11 @@ export default {
 
 	&__title {
 		margin-bottom: rem-calc(4);
+		display: inline;
```

Comment:
Wondering if this should be inline-block? I don't think the margin-bottom will do anything if it's inline

Reply:
Thanks for calling this out Ryan. I set it to inline, which allowed the superscript #1 to jump up on the same line as the headline text. But, yeah this does make the headline loose it's small bottom margin. Do you think there would be a problem with just adding that small margin-bottom to margin-top on the element below it?

Reply:
Probably fine adding margin top on the element below, but I think inline-block will let you have both the #1 on the same line and also the margin-bottom.

Reply:
I've tried making that update, but doesn't seem to be working. If I update the heading to `display: inline-block`, the 1 jumps to the next line. If I then alter the display settings on the <a>1</a>, to any within this available display options, I still get them displayed on two lines.